src/chrome/content/rules/Andrew-Munsell.xml

```diff
@@ -0,0 +1,9 @@
+<ruleset name="Andrew Munsell">
+
+	<target host="andrewmunsell.com" />
+	<target host="www.andrewmunsell.com" />
+
+	<rule from="^http://(?:www\.)?andrewmunsell\.com/"
+		to="https://www.andrewmunsell.com/" />
```

Comment:
Note that this one is specifically rewriting the non-www form to the www form. For those I've generally been fine with this style.

I will note that current style is to do regular capturing groups for better readability: `^http://(www\.)?andrewmunsell\.com/`